SharedSources/MediaLibraryModel/MediaLibraryBaseModel.swift

```diff
@@ -20,6 +20,8 @@ protocol MediaLibraryBaseModel {
     var indicatorName: String { get }
     var cellType: BaseCollectionViewCell.Type { get }
 
+    var medialibrary: VLCMediaLibraryManager { get }
+
```

Comment:
I fail to understand the motivation to expose the medialibrary. Could you elaborate on this?

Reply:
> I find a bit confusing that PR #212 and PR #213 are based on this PR.
> 
> I think PRs should be as independent as possible.

This is done on purpose so that you can check it out. You're still able to review just the last commit by selecting it in the diff

Reply:
I was thinking about this, a solution could be the following.

To avoid to expose the medialibrary, we could have a `createPlaylist` method in the models that can create a playlist.

 It could for example have a prototype like so:

`func createPlaylisty(_ name: String, filesIndexes: Set<IndexPath>? = nil)`

This would mean that the said model is responsible for the media addition in the playlist. That can help for cases such as `Genre` (it could add all media in the selected genres in a playlist).

Reply:
so I was going down that road for a bit until I realised that we will need to have the existing Playlists as well, to show them as selection options. A model editing another model doesn't feel right. Which means the controller needs to have access to a medialibrary if not through the model then by giving it the services.